.bazelci/presubmit.yml

```diff
@@ -538,17 +538,13 @@ tasks:
     platform: ubuntu2004
     working_directory: examples/crate_universe_local_path
     run_targets:
-      - "//:vendor_lazy_static_out_of_tree"
-    test_targets:
-      - "//..."
+      - "//:vendor_edit_test_out_of_tree"
   crate_universe_local_path_in_tree:
     name: Crate Universe Local Path In Tree
     platform: ubuntu2004
     working_directory: examples/crate_universe_local_path
     run_targets:
-      - "//:vendor_lazy_static_in_tree"
-    test_targets:
```

Comment:
Why delete `test_targets`?

Reply:
The run script now does the `bazel test` an part of an integration test, and it doesn't seem useful doing the test twice